docker-compose.yaml

```diff
@@ -1,224 +1,223 @@
-        ports:
-        # TODO repalce prot with env variable
-            - "8085:8080"
-
-    # Geoshop/Postgis
-    postgis:
-        image: postgis/postgis:14-3.2-alpine
-        volumes:
-          - ./geoshop/resources/ini-scripts:/docker-entrypoint-initdb.d/:ro
-          - ./geoshop/resources/postgis_data:/var/lib/postgresql/data:rw
-        environment:
-            LANG: 'en_US.utf8'
-            LC_COLLATE: 'en_US.utf8'
-            LC_CTYPE: 'en_US.utf8'
-        env_file: ./geoshop/.env
-        ports:
-            - "5432:5432"
-        healthcheck:
-            test: ["CMD-SHELL", "pg_isready -U geoshop"]
-            interval: 5s
-            timeout: 5s
-            retries: 5
-
-    # Mailhog
-    mailhog:
-        image: mailhog/mailhog
-        tty: true
-        ports:
-            - "1025:1025"
-            - "8025:8025"
-        volumes:
-            - ./mailhog:/home/mailhog/conf
-        entrypoint: MailHog -auth-file=/home/mailhog/conf/auth-users
-
-    # LDAP
-    openldap:
-        image: osixia/openldap:1.5.0
-        environment:
-            - LDAP_ORGANISATION=Extract
-            - LDAP_DOMAIN=extract.org
-            - LDAP_ADMIN_PASSWORD=extract
-            - LDAP_CONFIG_PASSWORD=extract
-            - LDAP_RFC2307BIS_SCHEMA=true
-            - LDAP_REMOVE_CONFIG_AFTER_SETUP=true
-            - LDAP_TLS_VERIFY_CLIENT=try
-        ports:
-            - "389:389"
-            - "636:636"
-
-    ldap-ad:
-        build: ./ldap-ad
-        volumes:
-            - ./ldap-ad/users.ldif:/ldap/users.ldif
-        ports:
-            - "10389:10389"
-
-    # QGIS
-    qgisserver:
-        image: camptocamp/qgis-server
-        volumes:
-            - ./qgis/project:/etc/qgisserver
-        environment:
-            - QGIS_PROJECT_FILE=/etc/qgisserver/world.qgs
-            - QGIS_SERVER_LANDING_PAGE_PROJECTS_DIRECTORIES=/etc/qgisserver
-        networks:
-            - qnetwork
-
-    qgisproxy:
-        depends_on:
-            qgisserver:
-                condition: service_started
-        build:
-            context: ./qgis
-        hostname: qgisproxy
-        container_name: qgisproxy
-        volumes:
-            - ./qgis/nginx.conf:/etc/nginx/nginx/conf:ro
-        ports:
-            - 8888:80
-        networks:
-            - qnetwork
-            - default
-
-    test:
-        profiles: [testing]
-        build: ./tests
-        networks:
-            - default
-        depends_on:
-            selenium:
-                condition: service_healthy
-            frontend:
-                condition: service_started
-
-    selenium:
-        profiles: [testing]
-        image: selenium/standalone-firefox
-        networks:
-            - default
-        ports:
-            - 4444:4444
-        healthcheck:
-            test: curl --fail http://localhost:4444/ || exit 1
-            interval: 10s
-            timeout: 10s
-            retries: 15
+  # Frontend
+  frontend:
```

Comment:
This is some FE for the extractor.
The naming is a bit unlucky. I marked a comment for the geoshop FE is further down.